### ISSUE TITLE
Error in run `itk-convert xxx.jpg to xxx.png`

### ISSUE THREAD

**Issue by Lionad-Morotar** [InsightSoftwareConsortium/itk-js#197]
![image](https://user-images.githubusercontent.com/26913795/53481681-af3e4d00-3ab8-11e9-910f-e79edeae424e.png)

* System Info *
Window 10, Node v10.15.1, NPM v6.8.0

**Comment by thewtex** [comment]
Thanks for the report.

If you build / test the package locally:

https://github.com/InsightSoftwareConsortium/itk-js/tree/master/examples/Node.js

i.e.

```
git clone https://github.com/InsightSoftwareConsortium/itk-js
cd itk-js/examples/Node.js
npm install
npm test
```

Does the test fail?

**Comment by Lionad-Morotar** [comment]
@thewtex Hi~ I tried your code but it still didn't work.

** error info ** 
the same as last error info: 
![image](https://user-images.githubusercontent.com/26913795/53539992-b1eb8180-3b4e-11e9-9e4d-e0e470d142ce.png)

** system info **
```
    System:
      OS: Windows 10 / 1803 / 17134.590
    Binaries:
      Node: 10.15.1 - C:\Program Files\nodejs\node.EXE
      npm: 6.8.0 - C:\Program Files\nodejs\npm.CMD
      python: 2.7.2
```

** extra **
I don't use nvm, so I use the latest nodejs. 
And can I provide other forms of help if I don't try to downgrade my nodejs?

**Comment by thewtex** [comment]
This is now addressed in `itk` 9.4.1 and `itk-convert` 1.1.1. Please upgrade and try again. Thanks again for the report :+1:

**Comment by Lionad-Morotar** [comment]
thank~
